packages/azure-themes/webpack.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = resources.createConfig(BUNDLE_NAME, IS_PRODUCTION, {
 
   output: {
     libraryTarget: 'var',
-    library: 'FabricAzureThemes',
+    library: 'FluentUIAzureThemes',
```

Comment:
this makes me realize, we should add/note this change in our release notes

Reply:
Yeah I might update that with all the new package names at the end